README.md

```diff
@@ -391,6 +391,7 @@ Badge | URL
 <img src="https://img.shields.io/badge/Redwood-FDF8F6?style=for-the-badge&logo=redwoodjs&logoColor=BF4722" /> | `https://img.shields.io/badge/Redwood-FDF8F6?style=for-the-badge&logo=redwoodjs&logoColor=BF4722`
 <img src="https://img.shields.io/badge/reveal%20js-F2E142?style=for-the-badge&logo=reveal.js&logoColor=000" /> | `https://img.shields.io/badge/reveal.js-F2E142?style=for-the-badge&logo=reveal.js&logoColor=000`
 <img src="https://img.shields.io/badge/rolup%20js-EC4A3F?style=for-the-badge&logo=rollup.js&logoColor=white" /> | `https://img.shields.io/badge/rolup.js-EC4A3F?style=for-the-badge&logo=rollup.js&logoColor=white`
+<img src="https://img.shields.io/badge/Robot%20Framework-000000?style=for-the-badge&logo=robot-framework&logoColor=white" /> | `https://img.shields.io/badge/Robot%20Framework-000000?style=for-the-badge&logo=robot-framework&logoColor=white`
```

Comment:
Your arrangement with `Alphabetical` order isn't correct. Please rearrange your badge before `Rolup JS`.

Reply:
@mrhrifat: Thanks for correcting me. Added `Robot Framework` to line 393 before `Rolup JS`